pandas/_version.py

```diff
@@ -101,10 +101,8 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
     if not dirname.startswith(parentdir_prefix):
         if verbose:
             print(
-                "guessing rootdir is '{root}', but '{dirname}' "
-                "doesn't start with prefix '{parentdir_prefix}'".format(
-                    root=root, dirname=dirname, parentdir_prefix=parentdir_prefix
-                )
+                f"""guessing rootdir is '{root}', but '{dirname}'
+                doesn't start with prefix '{parentdir_prefix}'"""
```

Comment:
don't use triple-quoted strings, it's not the same as two single-quoted strings.